aviatrix/resource_aviatrix_filebeat_forwarder.go

```diff
@@ -0,0 +1,136 @@
+package aviatrix
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-aviatrix/goaviatrix"
+)
+
+func resourceAviatrixFilebeatForwarder() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAviatrixFilebeatForwarderCreate,
+		Read:   resourceAviatrixFilebeatForwarderRead,
+		Delete: resourceAviatrixFilebeatForwarderDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"server": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Server IP",
+			},
+			"port": {
+				Type:        schema.TypeInt,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Port number",
+			},
+			"trusted_ca_file_path": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "Trusted CA file path",
+			},
+			"config_file_path": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "Configuration file path",
+			},
```

Comment:
Why file path and not file content?

Reply:
trusted_ca_file_path and config_file_path won't be returned by the backend.

Reply:
The backend does not return the file path either but that should not impact our choice of file path vs content

I thought we had discussed previously that using file content is better for customers since it makes working with remote data (files in s3 buckets for example) easier to work with.

Reply:
I see. Then the PostFileAPI needs to be modified a bit. 
Do we also need to maintain the content of trusted_ca_file?

Reply:
Not sure what you mean by "maintain the content" There is nothing special you need to do with the content, it will be saved to state after Create